proposals/0522-or-patterns.rst

```diff
@@ -1,12 +1,12 @@
-Or Patterns
+Or Patterns (Amended)
```

Comment:
```suggestion
Or Patterns
```

Reply:
I'm not sure if the meta data needs to be changed like that. Perhaps the committee will know what to do here?

Reply:
Usually not needed. We can mention the amending in the line with the acceptance date